SPECS/mariner-rpm-macros/mariner-rpm-macros.spec

```diff
@@ -12,23 +12,28 @@ Source2: default-hardened-cc1
 Source3: default-hardened-ld
 Source4: default-annobin-cc1
 Source5: macros.check
+Source6: macros.go-rpm
```

Comment:
Do we need this change in `1.0-dev`? It might save us some time and conflict resolution if we go straight to `dev`.